servers/rendering/renderer_rd/storage_rd/particles_storage.h

```diff
@@ -360,6 +360,7 @@ class ParticlesStorage : public RendererParticlesStorage {
 		virtual void set_code(const String &p_Code);
 		virtual bool is_animated() const;
 		virtual bool casts_shadows() const;
+		virtual RendererRD::MaterialStorage::ShaderData::CullMode get_cull_mode() const;
```

Comment:
We shouldn't expose this, instead we should read the cull mode from the shader directly.